data/operators.csv

```diff
@@ -418,3 +418,4 @@ Rose Telecom,ISP,signed + filtering,safe,54681,24936
 Bryan Barbolina trading as Cloudwebservices,cloud,signed + filtering,safe,213268,65065
 Hazelnet,ISP,signed + filtering,safe,200242,75603
 Parknet,ISP,signed + filtering,safe,197301,19561
+Datacamp Limited, transit, unsafe, 212238, 11971
```

Comment:
This line is missing a column for whether it signed its prefixes and whether it does filtering.

Based on the most recent data, it signed 82% of the prefixes it originates, so I'd incline to classify it as signed but not filtering.

```suggestion
Datacamp Limited,signed,transit,unsafe,212238,11971
```